ee/billing/test/test_quota_limiting.py

```diff
@@ -89,7 +90,8 @@ def test_quota_limit_feature_flag_not_on(self, patch_feature_enabled, patch_capt
         self.organization.save()
 
         time.sleep(1)
-        result = update_all_org_billing_quotas()
+        with self.assertNumQueries(2):
```

Comment:
small nit: This is one of the few places where I think it's worth snapshotting the actual query, not just the number. You can do:

```
with snapshot_postgres_queries_context(self)
```

which will snapshot only this bit, if for example there's lots of calls happening in the test and you only want snapshots for this bit

Reply:
oh good to know thanks!